mx.truffleruby/mx_truffleruby.py

```diff
@@ -59,7 +59,7 @@ class TruffleRubyLauncherBuildTask(mx.ArchivableBuildTask):
     def __init__(self, *args):
         mx.ArchivableBuildTask.__init__(self, *args)
 
-        self.jvm_args_file = join(root, 'tool', 'jvm_args.sh')
+        self.jvm_args_file = join(self.subject.suite.get_output_root(), 'jvm_args.sh')
```

Comment:
I think hardcoding this to `join(root, 'mxbuild')` might clearer about expectations, because if get_output_root() ever changes, it will break the launcher (or worse, use a stale version of the file).
Can you switch to that to make it obvious and ensure it stays in a well-known path?

Reply:
can be, it's hardcoded in the sh launcher so it might as well be hardcoded here. done